lib/i18n/backend/active_record.rb

```diff
@@ -74,6 +77,15 @@ def lookup(locale, key, scope = [], options = {})
             key = key[0..-2]
           end
 
+          if ActiveRecord.config.cache_translations
+            if @translations.nil? || @translations.empty?
+              self.send(:init_translations)
+            end
+
+            keys = ([locale.to_sym] + key.split(I18n::Backend::Flatten::FLATTEN_SEPARATOR)).map(&:to_sym)
```

Comment:
The first `to_sym` is obsolete as we call `to_sym` in the end

```suggestion
            keys = ([locale] + key.split(I18n::Backend::Flatten::FLATTEN_SEPARATOR)).map(&:to_sym)
            
```